src/neptune/api/operations.py

```diff
@@ -0,0 +1,200 @@
+#
```

Comment:
For every API operation we should add unittests for serialization like we done for `models`.